script.js

```diff
@@ -185,4 +185,35 @@ feature4.addEventListener('click', () => {
 
 IconClose4.addEventListener('click', () => {
   popup4.classList.toggle('active');
-});
\ No newline at end of file
+});
+
+/* Contact form */
+const ContactForm = document.getElementById('contact-form');
+const email = document.getElementById('email');
+const nameF = document.getElementById('name');
+const message = document.getElementById('message');
+const errorMsg = document.getElementById('errormsg');
+function contactForm(event) {
+  if (email.value === '') {
+    errorMsg.style.display = 'block';
+    errorMsg.innerHTML = 'Please enter your email';
+    event.preventDefault();
+  } else if (
+    email.value !== email.value.toLowerCase()) {
+    errorMsg.innerHTML = 'Please enter you email in Lowercase ';
```

Comment:
- [ ] Nice work implementing the validation :clap: However it is required in the project requirement that the form shouldn't be sent when the email is not lower case. Kindly check it 
**HIT:** Add `event.preventDefault()` when checking if the email is lower case